### DISCUSSION TITLE
Allure can't report info on parts of multipart HTTP-request.

### DISCUSSION THREAD

**Discussion by ikonovi** [allure-framework/allure2#2360] [Ideas & Feedback]
Hi All! 
I use Allure with Java and Rest-Assured. My test scenarios send  a few POST-requests of multipart type by Rest Assured library. When I generate Allure report, I can see header "`Content-Type: multipart/form-data`" in the attachment that holds info about request. The problem is that there is no info about parts itselve of multipart request. 
Could you please learn the Allure to put info about parts of multipart request in report as well?

**Comment by MahmoudAhmedKhalifa** [comment]
Did you reach a solution? @ikonovi
